universal-application-tool-0.0.1/app/services/applicant/ApplicantService.java

```diff
@@ -0,0 +1,28 @@
+package services.applicant;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.concurrent.CompletionStage;
+import models.Applicant;
+import services.ErrorAnd;
+
+/** Defines the interface facade for Applicant service */
+interface ApplicantService {
+
+  /**
+   * Performs a set of updates to the applicant's ApplicantData. Updates are atomic i.e. if any of
+   * them fail validation none of them will be written. programId is used to construct the {@code
+   * ReadOnlyApplicantService} provided in the return value.
+   */
+  CompletionStage<ErrorAnd<ReadOnlyApplicantService, UpdateError>> update(
+      long applicantId, long programId, ImmutableSet<Update> updates);
```

Comment:
issue: I don't see the `Update` class in this PR. Should we add it or is it fine to add it later?

Reply:
Oh I need to add it, thanks!

Reply:
done